package.json

```diff
@@ -58,10 +58,10 @@
     "stylelint-order": "^0.8.1",
     "stylelint-scss": "^3.1.0",
     "stylelint-suitcss": "^3.0.0",
-    "terra-clinical": "git://github.com/cerner/terra-clinical.git#terra-clinical-onset-picker@2.7.0",
-    "terra-core": "git://github.com/cerner/terra-core.git#terra-tag@1.11.0",
+    "terra-clinical": "git://github.com/cerner/terra-clinical.git#terra-clinical-onset-picker@2.8.0",
+    "terra-core": "git://github.com/cerner/terra-core.git#terra-toggle@2.13.0",
```

Comment:
toggle?

Reply:
Based on how our mono-repos work, each release contains all the code for all the other components, so I just pull in the 'latest' release tag, whichever one that might be. So for terra core, the top release is terra toggle. https://github.com/cerner/terra-core/releases